src/plone/restapi/indexers.py

```diff
@@ -55,59 +59,48 @@ def __call__(self, value):
         return result
 
 
+@implementer(IBlockSearchableText)
+@adapter(IBlocks, IBrowserRequest)
+class SlateTextIndexer:
+    """Searchable Text indexer for Slate blocks."""
+
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
+
+    def __call__(self, block):
+        block = block or {}
+        # BBB compatibility with slate blocks that used the "plaintext" field
```

Comment:
plaintext is not current practice?

Reply:
I was surprised as well, but it seems it was agreed searchableText should be the way to go.